src/Microsoft.Fx.Portability.Reports.Html/CompatibilitySummaryModel.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Fx.Portability.ObjectModel;
+using Microsoft.Fx.Portability.Reporting.ObjectModel;
+using System.Collections.Generic;
+
+namespace Microsoft.Fx.Portability.Reports.Html
+{
+    public class CompatibilitySummaryModel
+    {
+        public IDictionary<BreakingChange, IEnumerable<MemberInfo>> Breaks { get; }
+        public AssemblyInfo Assembly { get; }
+        public ReportingResult ReportingResult { get; }
+        public int LoopCounter { get; }
```

Comment:
What does LoopCounter represent? Is it the number of issues encountered?

Reply:
I created the property while refactoring the report into partial views. Turns out it's unneeded. The counter value was used to ensure unique ids for compatibility summaries so the script which collapses/expands them can work. Now that summaries are generated in a partial view, we can just generate GUIDs there instead of passing a unique identifier in via the model.